### PR TITLE
Rename python driver

### PR DESCRIPTION
**Pull request by gabor-boros**
## Description
The package renamed to `rebirthdb` from `rethinkdb`. Related to the rename, every reference in the code has been changed. Also added improvements to the `Makefile`'s clean command to remove trash what `tox` leaves in the package. Extended README to include build status, and some note about the package.

*NOTE*
The mentioned pypi package name is not exists yet.

### CODE REVIEW

File: rebirthdb/asyncio_net/net_asyncio.py [hunk 1]
```diff
@@ -21,9 +21,15 @@
...
 
+<<<<<<< Updated upstream:rethinkdb/asyncio_net/net_asyncio.py
 from rethinkdb import ql2_pb2
 from rethinkdb.errors import ReqlAuthError, ReqlCursorEmpty, ReqlDriverError, ReqlTimeoutError, RqlCursorEmpty
 from rethinkdb.net import Connection as ConnectionBase, Cursor, Query, Response, maybe_profile
+=======
+from rebirthdb import ql2_pb2
+from rebirthdb.errors import ReqlAuthError, ReqlCursorEmpty, ReqlDriverError, ReqlTimeoutError, RqlCursorEmpty
+from rebirthdb.net import Connection as ConnectionBase, Cursor, Query, Response, maybe_profile
+>>>>>>> Stashed changes:rebirthdb/asyncio_net/net_asyncio.py
```
[the remark this answers is not in the document]
Comment (gabor-boros) [reply rebirthdb/asyncio_net/net_asyncio.py]: Wow, It remained there. Give me 2 minutes.

[the remark this answers is not in the document]
Comment (gabor-boros) [reply rebirthdb/asyncio_net/net_asyncio.py]: By the way, I'm wondering how the IDE did not found it as a merge conflict.

[the remark this answers is not in the document]
Comment (grandquista) [reply rebirthdb/asyncio_net/net_asyncio.py]: This file is conditionally loaded. That might explain.

**Comment by gabor-boros** [comment]
Requested changes are adressed

**Pull request merged**